src/components/NewsletterSignup/NewsletterSignup.tsx

```diff
@@ -40,39 +37,49 @@ interface NewsletterSignupProps {
   id?: string;
   /** Toggles the invalid state for the email field. */
   isInvalidEmail?: boolean;
-  /** Value to determine the section color highlight. The default is set to "blogs" as it uses the
-   * "ui.border.deafult" color.
+  /** Value to determine the section color highlight.
    */
-  newsletterSignupType?: SectionTypes;
+  highlightColor?: HighlightColorTypes;
   /** A handler function that will be called when the form is submitted. */
   onSubmit: (event: React.FormEvent<any>) => void;
   /** A handler function that will be called when the text input changes. */
   onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  /** Gives option to remove the default Privacy Link if a custom one is provided
+   * NOTE: A Privaly Policy link should always be included.
+   */
+  showDefaultPrivacyLink?: boolean;
   /** Link to the relevant privacy policy page. */
   privacyPolicyLink?: string;
-  /** Optional string value used to set the text for a `Heading` component, or
+  /** Sets the text for a `Heading` component, or
    * a DS Heading component that can be passed in.
    */
-  title?: JSX.Element | string;
+  title: JSX.Element | string;
   /** The value of the email text input field. */
   valueEmail?: string;
   /** Used to specify what is displayed in the component form/feedback area. */
   view?: NewsletterSignupViewType;
 }
 
+export type HighlightColorTypes =
```

Comment:
Hey yes I know about that syntax. I chose the other way because I personally like better when the type name is shown in the storybook arguments table rather than the list: 
![Showing all possible colors](https://github.com/NYPL/nypl-design-system/assets/70068858/28bbbc84-5822-473d-bdad-2f499f724413)
vs
![Only showing the type](https://github.com/NYPL/nypl-design-system/assets/70068858/ef24e50f-e7d1-4b57-a8ac-9bd77ce54e29)

Reply:
Updated it though :)

Reply:
Yes, it is might nicer and clearer with all the values listed down :\